ts/api/backend.ts

```diff
@@ -241,6 +310,9 @@ export function BackendClient(
       createOrUpdateInstallationT,
       options
     ),
-    logout: createFetchRequestForApi(logoutT, options)
+    logout: createFetchRequestForApi(logoutT, options),
+    getVerificaRpt: createFetchRequestForApi(verificaRptT, options),
```

Comment:
one thing worth noting, for verifica and attiva we will need to use a different `fetchApi` than the rest of the backend client - due to very high response times (in the order of 10-15 seconds) we need to provide a different `fetch` with higher timeout and, I would say, different retry semantics (I'm not sure we even need a retry policy due to the behavior of attiva) - cc @LsKoder